File: bimestre1/aula_04-16/04-fahrenheit.go
package main
import "fmt"

// Escreva um programa que converta temperaturas de Fahrenheit para Celsius,
// seguindo a equação 𝐶 = ((𝐹−32)×5)/9
func main() {
	var f int

	fmt.Print("Informe a temperatura em ºF: ")
	fmt.Scanf("%d", &f)

	var c int = ((f-32)*5)/9

	fmt.Printf("%dºF = %dºC\n", f, c);
}

File: bimestre1/aula_05-07/04-fahrenheit.hs
-- escreva um programa que converta temperaturas de fahrenheit para celsius, seguindo a equação c = ((f-32)*5)/9
fahrenheittocelsius :: Int -> Int
fahrenheittocelsius x = ((x-32)*5) `div` 9

main :: IO ()
main = print (fahrenheittocelsius 50)
